alembic/env.py

```diff
@@ -0,0 +1,81 @@
+from logging.config import fileConfig
```

Comment:
The import statement for `fileConfig` is missing a proper import path. Ensure that `fileConfig` is imported from the correct module, which is `from logging.config import fileConfig`.